### PR TITLE
PT-314, PT-316 | Route for create enrolment

### PR DESCRIPTION
**Pull request by jorilindell**
1. New page to create an enrolment
2. Event info on create enrolment page

### CODE REVIEW

File: src/domain/enrolment/CreateEnrolmentPage.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import { useEventQuery } from '../../generated/graphql';
+import { Router } from '../../i18n';
+import Container from '../app/layout/Container';
+import PageWrapper from '../app/layout/PageWrapper';
+import { ROUTES } from '../app/routes/constants';
+import NotFoundPage from '../notFoundPage/NotFoundPage';
+import styles from './enrolmentPage.module.scss';
+import EventInfo from './eventInfo/EventInfo';
+
+const CreateEnrolmentPage = (): ReactElement => {
```
Comment (Sartonon) [inline src/domain/enrolment/CreateEnrolmentPage.tsx:16]: Would React.FC be more convenient in component definitions?
const CreateEnrolmentPage: React.FC = () => { ?

File: src/domain/enrolment/eventInfo/EventInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import { EventFieldsFragment } from '../../../generated/graphql';
+import useLocale from '../../../hooks/useLocale';
+import getLocalisedString from '../../../utils/getLocalisedString';
+import styles from './eventInfo.module.scss';
+
+interface Props {
+  event: EventFieldsFragment;
+}
+
+const EventInfo = ({ event }: Props): ReactElement => {
```
Comment (Sartonon) [inline src/domain/enrolment/eventInfo/EventInfo.tsx:12]: Would React.FC be more convenient in component definitions?:
const EventInfo: React.FC<Props> = ({ event }) => { ?

File: src/domain/enrolment/CreateEnrolmentPage.tsx (hunk 1 above)
Comment (jorilindell) [reply src/domain/enrolment/CreateEnrolmentPage.tsx]: Fixed

File: src/domain/enrolment/eventInfo/EventInfo.tsx (hunk 2 above)
Comment (jorilindell) [reply src/domain/enrolment/eventInfo/EventInfo.tsx]: Fixed

**Pull request merged**